jobs/kola-azure.Jenkinsfile

```diff
@@ -136,29 +130,26 @@ timeout(time: 75, unit: 'MINUTES') {
             // Since we don't have permanent images uploaded to Azure we'll
             // skip the upgrade test.
             try {
-                def azure_subscription = shwrapCapture("jq -r .subscriptionId \${AZURE_KOLA_TESTS_CONFIG_AUTH}")
+                def azure_subscription = shwrapCapture("jq -r .subscription \${AZURE_KOLA_TESTS_CONFIG}")
```

Comment:
guess this `subscriptionId` change to `subscription` was intentional.

Reply:
yes, the file format changed. See the first commit in this patch set.

Reply:
yes got it.